checker/build.xml

```diff
@@ -1508,6 +1508,8 @@
          <env key="LT_BIN" value="${LT_BIN}"/>
       </exec>
       <move file="jdk/jdk.jar" tofile="jdk/${jdkName}"/>
+        <!-- Copy jars to dist. If dist doesn't exist then the copy task creates it.-->
+        <copy file="jdk/${jdkName}" tofile="dist/${jdkName}"/>
```

Comment:
Target `dist-nobuildjdk` at the moment also copies around jdks. Those copies be removed now, right?